lang-python/ecdsa/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=ecdsa
+PKGSEC=python
+PKGDEP="python-3 six"
+BUILDDEP="setuptools-python3"
+PKGDES="Implementation of ECDSA in Python"
+
+NOPYTHON2=1
+ABSPLITDBG=0
```

Comment:
This is an architecture-agnostic package, replace this line with:

```
ABHOST=noarch
```

To indicate as such.